src/test/java/digkins/services/ArtifactsServiceTests.java

```diff
@@ -0,0 +1,58 @@
+package digkins.services;
+
+import com.offbytwo.jenkins.JenkinsServer;
+import com.offbytwo.jenkins.client.JenkinsHttpClient;
+import com.offbytwo.jenkins.model.*;
+import com.redhat.digkins.services.ArtifactsService;
+import org.junit.Test;
+
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.util.Arrays;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Matchers.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+public class ArtifactsServiceTests {
+  private String jobLocation = "http://localhost/job/artifact/";
```

Comment:
I would avoid field initialisation, instead I find more convenient doing it inside @Before method